<!DOCTYPE html>
<html>
  <head>
    <title>Tracking the pandemic – Mihály Koltai – personal website</title>

        <meta charset="utf-8" />
    <meta content='text/html; charset=utf-8' http-equiv='Content-Type'>
    <meta http-equiv='X-UA-Compatible' content='IE=edge'>
    <meta name='viewport' content='width=device-width, initial-scale=1.0, maximum-scale=1.0'>

    
    <meta name="description" content="Graphs and notes on the science and economics of COVID19" />
    <meta property="og:description" content="Graphs and notes on the science and economics of COVID19" />
    
    <meta name="author" content="Mihály Koltai" />

    
        






    


    
    <meta property="og:title" content="Tracking the pandemic" />
    <meta property="twitter:title" content="Tracking the pandemic" />
    


    <!--[if lt IE 9]>
      <script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script>
    <![endif]-->

    <link rel="stylesheet" type="text/css" href="/style.css" />
    <link rel="alternate" type="application/rss+xml" title="Mihály Koltai - personal website" href="/feed.xml" />

    <!-- Created with Jekyll Now - http://github.com/barryclark/jekyll-now -->
    <!-- Begin Jekyll SEO tag v2.5.0 -->
<title>Tracking the pandemic | Mihály Koltai</title>
<meta name="generator" content="Jekyll v3.7.3" />
<meta property="og:title" content="Tracking the pandemic" />
<meta property="og:locale" content="en_US" />
<meta name="description" content="Graphs and notes on the science and economics of COVID19" />
<meta property="og:description" content="Graphs and notes on the science and economics of COVID19" />
<link rel="canonical" href="http://localhost:4000/covid-visualization/" />
<meta property="og:url" content="http://localhost:4000/covid-visualization/" />
<meta property="og:site_name" content="Mihály Koltai" />
<meta property="og:type" content="article" />
<meta property="article:published_time" content="2020-04-09T00:00:00+02:00" />
<script type="application/ld+json">
{"description":"Graphs and notes on the science and economics of COVID19","@type":"BlogPosting","url":"http://localhost:4000/covid-visualization/","headline":"Tracking the pandemic","dateModified":"2020-04-09T00:00:00+02:00","datePublished":"2020-04-09T00:00:00+02:00","mainEntityOfPage":{"@type":"WebPage","@id":"http://localhost:4000/covid-visualization/"},"@context":"http://schema.org"}</script>
<!-- End Jekyll SEO tag -->


  </head>

  <body>
    <div class="wrapper-masthead">
      <div class="container">
        <header class="masthead clearfix">
          <a href="/" class="site-avatar"><img src="" /></a>

          <div class="site-info">
            <h1 class="site-name"><a href="/">Mihály Koltai</a></h1>
         <!--    <p class="site-description">personal website</p> -->
          </div>

          <nav>
            <a href="/">about</a>
            <a href="/papers">publications</a>
            <a href="/blog">maths&bio</a>
            <a href="/ecopol">eco/pol</a>
            <a href="/other">other</a>
          </nav>
        </header>
      </div>
    </div>

    <div id="main" role="main" class="container">
      <article class="post">
  <h1>Tracking the pandemic</h1>

  <div class="entry">
    <p>This post includes graphs of the dynamics of the COVID19 pandemic, as well as notes that I am making on the science (mostly epidemiologic modeling), economics and politics of the pandemics. As of early April (9th of April) the notes are rather messy, I plan to clean them up with time.</p>

<p>There are two main interests that go through the notes. First, I am interested in mathematical modeling of how the epidemic is spreading and the efforts to slow it down or suppress it. Second, I am interested in structural analysis of how it might change society, especially on the medium/long-term.</p>

<h2 id="table-of-contents">Table of contents</h2>

<ol>
  <li><a href="#1-data-visualization">Data visualization</a></li>
  <li><a href="#2-science-and-modeling">Science and modeling</a></li>
  <li><a href="#3-economics-and-politics">Economics and politics</a></li>
</ol>

<h2 id="1-data-visualization">1. Data visualization</h2>

<p>The data source for all these plots is the central database by <a href="https://github.com/CSSEGISandData/COVID-19/raw/master/csse_covid_19_data/csse_covid_19_time_series">Johns Hopkins University</a>.</p>

<h3 id="total-number-of-cases-by-date-logarithmic-y-axis">Total number of cases by date (logarithmic y-axis)</h3>

<p>This is showing the cumulative number of confirmed COVID19 cases in the major European countries, the US and some of the major Asian countries. The numbers are plotted by date.
The 5 biggest European countries by population (Germany, UK, France, Italy and Spain) have a combined population almost exactly equal to the US, so it is useful to plot them also as a group, this is shown by ‘<em>IT_ES_FR_DE_UK’</em>.
The insets show European countries plus the US and Asian countries, respectively.
The dashed blue lines on the left inset show the dates:</p>
<ul>
  <li>20/February: exponential increase of cases starts in Italy</li>
  <li>26/February: exponential increase of cases starts in France</li>
  <li>09/March: countrywide lockdown in Italy</li>
  <li>16/March: countrywide lockdown in France (announced, effective from 17th noon)</li>
</ul>

<p><img src="/images/covid/west_vs_asia_w_china.png" alt="_config.yml" /></p>

<h3 id="total-number-of-cases-aligned-logarithmic-y-axis">Total number of cases, aligned (logarithmic y-axis)</h3>

<p>Same as the previous plot, but countries are aligned to start from the same number of cases as Italy had on the 26th of February (or the day they had the closest value).</p>

<p><img src="/images/covid/cases_cntrs_tracking_italy_log.png" alt="_config.yml" /></p>

<h3 id="total--deaths-aligned-from-same-starting-point-y-logscale">Total # deaths, aligned from same starting point (y logscale)</h3>

<p>Same as the previous plot, but for cumulated number of fatalities.</p>

<p><img src="/images/covid/deaths_cntrs_tracking_italy_log.png" alt="_config.yml" /></p>

<h3 id="number-of-new-cases-in-europe-us-and-asia-logarithmic-y-axis">Number of new cases in Europe, US and Asia (logarithmic y-axis)</h3>

<p>Plotting the number of new cases by date, for the same Western (+Russia) and Asian countries.</p>

<p><img src="/images/covid/newcases_west_vs_asia_w_china.png" alt="_config.yml" /></p>

<h3 id="number-of-new-cases-in-europe-and-the-us">Number of new cases in Europe and the US</h3>

<p>Number of new confirmed cases by date for Western countries and Russia only. The dashed blue lines show the imposition of the lockdowns in Italy (02/March, 09/March) and France (16/March).</p>

<p><img src="/images/covid/newcases_west.png" alt="_config.yml" /></p>

<h3 id="number-of-daily-deaths-in-europe-and-the-us">Number of daily deaths in Europe and the US</h3>

<p>The dashed blue lines show the imposition of the lockdowns in Italy (02/March, 09/March) and France (16/March).</p>

<p><img src="/images/covid/newdeaths_west.png" alt="_config.yml" /></p>

<h3 id="growth-rate-of-cases-on-total-number-of-cases">Growth rate of cases on total number of cases</h3>

<p>This plot shows the percentage (fractional) growth of the total number of cases, as a function of the total number, with time going from left to right. The growth rate is averaged over a sliding window of three days.</p>

<p><img src="/images/covid/growth_rate_cases_china_korea_west_cutoff2000.png" alt="_config.yml" /></p>

<h3 id="growth-rate-of-deaths-on-the-total-number">Growth rate of deaths on the total number</h3>

<p>As previous plot, but for the number of fatalities. The growth rate is averaged over a sliding window of three days.</p>

<p><img src="/images/covid/growth_rate_deaths_china_korea_west_cutoff60.png" alt="_config.yml" /></p>

<h4 id="why-italy-has-a-very-high-case-fatality-rate-cfr">Why Italy has a very high case fatality rate (CFR)?</h4>
<p><a href="https://medium.com/@andreasbackhausab/coronavirus-why-its-so-deadly-in-italy-c4200a15a7bf">% of old people/all cases much higher</a><br />
Reason for many older ppl infected: <a href="https://twitter.com/kuhnmo/status/1238421146837684224">20% of Italians 30-49 still live at home</a><br />
+ testing policy was changed late March bc of capacity limits, and only testing severe cases, so the number of confirmed cases is smaller, so ratio of death/confirmed cases grows.</p>

<div style="page-break-after: always;"></div>

<h2 id="2-science-and-modeling">2. Science and modeling</h2>

<h4 id="school-closures-event-cancellations-and-the-mesoscopic-localization-of-epidemics-in-networks-with-higher-order-structure"><a href="https://arxiv.org/pdf/2003.05924.pdf">School closures, event cancellations, and the mesoscopic localization of epidemics in networks with higher-order structure</a></h4>
<p>higher-order struct leads to mesoscopic localization (concentration into substructures)
Individuals classified by # of contacts:<br />
<script type="math/tex">S_k</script>: % susceptible with k contacts<br />
<script type="math/tex">I_k</script>: % infected with k contacts<br />
Clique prevalence: average fraction of nodes within cliques of size <em>n</em>.</p>

<p><img src="/images/covid/clique%20prevalence.PNG" alt="_config.yml" /></p>

<p>What does lower coupling mean?
There are two distributions that characterize the network:<br />
<script type="math/tex">g_m</script>: membership of nodes. <script type="math/tex">S_m</script> is % of <em>S</em> with number <em>m</em> of memberships in higher order structures.  <script type="math/tex">g_m \sim m^{-\gamma}</script><br />
p_n: distribution of clique size <em>n</em>. <script type="math/tex">p_n \sim n^{-\gamma}</script></p>

<p>If edges are removed, in the localized network there is a critical size n_c for cliques under which if enough edges are removed, then there are no infections anymore in these cliques.
Also, in localized networks global prevalence falls off much more sharply with stronger intervention, whereas effect is linear in ‘delocalized’ networks.</p>

<p>Variables are defined by master equations. This goes down after a week.
With active case finding the peak is lower and earlier.
With higher transmission rate this gets higher, converging to ~0.9.
But if structures are highly coupled (a), clique prevalence ~same for n [20,100].
With lower coupling (b) there is <em>mesoscopic localization</em>: global prevalence can be very low, but within cliques, much higher % of infection.</p>

<h4 id="interactive-modeling-httpsalhillshinyappsiocovid19seir">Interactive modeling: https://alhill.shinyapps.io/COVID19seir/</h4>
<p>This is an ODE model assuming complete homogeneity. Three <strong>I</strong> populations with different rates of transmission.</p>

<h4 id="the-contribution-of-pre-symptomatic-transmission-to-the-covid-19-outbreak"><a href="https://cmmid.github.io/topics/covid19/control-measures/pre-symptomatic-transmission.html">The Contribution of Pre-symptomatic Transmission to the COVID-19 Outbreak</a></h4>
<p>Estimates that ~25% of transmission in Shenzhen was asymptomatic.</p>

<h4 id="temporal-variation-in-transmission-during-the-covid-19-outbreak-in-italy"><a href="https://cmmid.github.io/topics/covid19/current-patterns-transmission/italy-time-varying-transmission.html">Temporal variation in transmission during the COVID-19 outbreak in Italy</a></h4>
<p>Estimates on reproduction numbers in regions of Italy. Started from 3-4, falling towards 2.</p>

<h4 id="shiny-app-for-contact-tracing-httpsbdi-pathogensshinyappsiocovid-19-transmission-routes">Shiny app for contact tracing: https://bdi-pathogens.shinyapps.io/covid-19-transmission-routes</h4>

<h4 id="substantial-undocumented-infection-facilitates-the-rapid-dissemination-of-novel-coronavirus-sars-cov2"><a href="https://science.sciencemag.org/content/sci/early/2020/03/13/science.abb3221.full.pdf">Substantial undocumented infection facilitates the rapid dissemination of novel coronavirus (SARS-CoV2)</a></h4>

<p>Estimates from a model + Bayesian inference.
Main findings (these are max probability estimates):</p>
<ul>
  <li>86% of infections were from undocumented cases in China <em>before travel ban of 23/january</em>.</li>
  <li>undocumented cases had 55% (mu*beta) transmission rate of documented ones (beta)</li>
  <li>undocumented cases were the source of 79% of documented cases</li>
</ul>

<p>Model: dynamics of infections among 375 Chinese cities.
Spatial dynamics captured by # of passengers from city <em>i</em> to <em>j</em>.
Variables for each city: <script type="math/tex">S_i, E_i, I_i^r, I_i^u</script> (undocumented).
Simulations by ‘iterated filter-ensemble adjustment Kalman filter (IF-EAKF) framework’.<br />
Inferred parameters:</p>
<ul>
  <li>Z: average latent period</li>
  <li>D: average duration of infection</li>
  <li><script type="math/tex">\mu</script>: transmission reduction factor for undocumented</li>
  <li>beta: transmission rate for documented</li>
  <li>alpha: fraction of documented infections</li>
  <li>theta: travel multiplicative factor</li>
</ul>

<p>Reporting delay is included (from Gamma distribution).
Best-fitting parameter values identified by maximum likelihood.
They could distinguish between mu-alpha value pairs with different individual values but same product. They did simulations with all parameter value pairs (within certain range) and .
Delay of reporting crucial, this needs to be speeded up to stop the spread.</p>

<p><img src="/images/covid/science_article_undocum_cases.PNG" alt="_config.yml" /></p>

<p><a href="https://science.sciencemag.org/content/sci/suppl/2020/03/13/science.abb3221.DC1/abb3221_Li_SM.pdf">SI</a> contains equations.
System of coupled ODEs: S_i is susceptibles in city <em>i</em>.
All terms normalised by total population (in that city).
D: <duration infection="">  
Then effective reproduction number $$R_e=\alpha \beta D + (1-\alpha)\mu \beta\ D$$</duration></p>

<p>System of ODEs are then solved stochastically: each RHS term is sampled from a Poisson distribution. So we take the current value of the variable and then sample from a Poisson distribution (the deterministic value is the lambda of the Poisson). This is for each RHS term!
Initial prior ranges for parameters are from Latin Hypercube Sampling.
Initial values for populations similarly sampled (eg. initial exposed).</p>

<p>Then the simulation results are mapped into reported cases by a separate delay model.
Delay is from a Gamma distribution approximated from data.
Movements between cities estimated from mobile apps: Tencent, Mobility Index from Baidu.
‘Iterated’ filtering to estimate model parameters.</p>

<h4 id="wolfram-mathematica-notebook-on-epidemic-modeling"><a href="https://community.wolfram.com/groups/-/m/t/1896178">Wolfram Mathematica notebook on epidemic modeling</a></h4>
<p>https://www.twitch.tv/videos/569280246</p>

<h4 id="necsi-slack-channel-for-modeling"><a href="https://app.slack.com/client/T2DRR8UNB/CV19X1PKJ/thread/CUTSF2Q6R-1584495843.398200">NECSI Slack channel for modeling</a></h4>

<h4 id="imperial-impact-of-non-pharmaceutical-interventions-npis-to-reduce-covid19-mortality-and-healthcare-demand"><a href="https://www.imperial.ac.uk/media/imperial-college/medicine/sph/ide/gida-fellowships/Imperial-College-COVID19-NPI-modelling-16-03-2020.pdf">Imperial: Impact of non-pharmaceutical interventions (NPIs) to reduce COVID19 mortality and healthcare demand</a></h4>

<p>2 strategies:
1) mitigation: can reduce peak healthcare demand by 2/3 and deaths by 50%. Still 100s of 1000s of deaths
2) Suppression: preferable for countries that can achieve. Means social distancing for <em>whole</em> of the population. Need to be maintained until vaccine found, as there is rebound if interventions are targeted.</p>

<p>Individual-based simulation model. Mixing in different environments. Schools double per capita rate of contacts. Incubation time 5 days. Infectiousness starts before symptoms. R0: between 2 and 2.6. Individual infectiousness varies (gamma distribution).
Symptomatic cases self-isolate w 1 day delay. 5 day delay to hospitalisation.</p>

<p>Results:</p>
<ul>
  <li>no measures: peak after 3 months, 80% of US/UK population infected. 30-time more demand for critical care beds than available. 2.2M dead in US, 0.5M dead in UK.</li>
  <li>mitigation: for 7 different levels of mitigation they make estimates for the reduction in peak bed demand and total deaths. For two values of R0 (2.2, 2.4) and 4 values of ‘triggers’ (cumulative ICU cases). Trigger means when policies (home quarantine etc) are switched on/off.</li>
  <li>suppression: all measures combined can keep ICU bed demand below UK surge bed capacity (5K). Adaptive triggering of suppression: on-off with some thresholds for # of ICU cases.
Even with most comprehensive suppression strategy in 2 yrs abt 30K deaths in UK.
Higher ON trigger and/or lower OFF -&gt; more deaths etc.</li>
</ul>

<p><a href="https://www.pnas.org/content/pnas/105/12/4639.full.pdf">Model used by Imperial from 2008</a></p>

<h4 id="necsi-response-to-imperial-paper-ferguson-et-al-impact-of-non-pharmarceutical"><a href="https://static1.squarespace.com/static/5b68a4e4a2772c2a206180a1/t/5e70eb32b16229792eb14836/1584458547530/ReviewOfFergusson.pdf">NECSI response to Imperial paper (Ferguson et al) ‘Impact of non-pharmarceutical…’</a></h4>

<p>They agree with Ferguson that suppression is needed to bring down R<sub>0</sub> below 1.
But they disagree w Ferguson claim that 2nd outbreak unavoidable. It’s not because, during lockdown infectious ppl are identified and their contacts isolated.
Ferguson calculate new infections as fcn of infected % and immunity, but not the phase of trajectory (ergodicity).
Not true that resurgence starts from even 1 case, China data showed <em>only 5% of close contacts of infected tested positive</em>. Small outbreaks can be controlled by testing.
Ferguson ignores super-spreader events by ignoring ‘fat tail distribution of contagion’.
But cutting off fat tail by banning events reduced R0.
Ferguson use SIR ODE-model, not appropriate bc they <em>ignore</em>:</p>
<ul>
  <li>local dynamics</li>
  <li>non-normal distrib. of infection events per person (superspreaders)</li>
  <li>dynamic or stochastic values for parameters (social response)</li>
</ul>

<p>But Imperial model has different environments and it is agent-based, so maybe this is not fair.
Debate: https://twitter.com/yaneerbaryam/status/1239936951823843328</p>

<h4 id="the-hammer-and-the-dance"><a href="https://medium.com/@tomaspueyo/coronavirus-the-hammer-and-the-dance-be9337092b56">The hammer and the dance</a></h4>

<p>Uses simple SIR model to argue why suppression is needed, as mitigation leads to enormous death toll (some estimates on side-effects).
Suppression buys time for producing equipment, maybe drugs.</p>

<h4 id="beyond-contact-tracing-community-based-early-detection-for-ebola-response-plos-currents-outbreaks-may-19-2016"><a href="https://necsi.edu/beyond-contact-tracing">Beyond contact tracing: Community-based early detection for Ebola response, PLoS Currents Outbreaks (May 19, 2016)</a></h4>

<p>Stochastic model by NECSI.</p>

<h4 id="community-prevention"><a href="https://static1.squarespace.com/static/5b68a4e4a2772c2a206180a1/t/5e737b95403f772d8ce0e04a/1584626591711/CommunityPrevention.pdf">Community prevention</a></h4>

<p>NECSI model. R<sub>*</sub>: community transmission rate.
i_0_c: stochastic factor of initial foothold the virus gains in a community
i_c(t): # active infections in community <em>c</em><br />
If infection not contained, <script type="math/tex">exp(r_c t)</script><br />
After <script type="math/tex">T_c</script> community implements aggressive suppression policies.
Infections decay as <script type="math/tex">exp(-t/\tau_c)</script><br />
After intervention # of infections is:<br />
<script type="math/tex">i_c(t)=( i_c^0 exp(r_c T_c) ) exp(-(t-T_c)/\tau_c)</script><br />
Community transmission <script type="math/tex">\sim i_c(t) p(travel)</script><br />
Poisson process, average number of infected communities can be estimated by integrating the two sub-processes (until and after <script type="math/tex">T_c</script>).<br />
Calculate expected number of other communities infected by community <script type="math/tex">c</script>.<br />
It’s assumed growth within one community is exponential until <script type="math/tex">T_c</script> (intervention), then exponential decay after <script type="math/tex">T_c</script>.</p>

<h4 id="social-contacts-and-mixing-patterns-relevant-to-the-spread-of-infectious-diseases"><a href="https://journals.plos.org/plosmedicine/article?id=10.1371/journal.pmed.0050074">Social Contacts and Mixing Patterns Relevant to the Spread of Infectious Diseases</a></h4>

<p>Estimates for mixing patterns.</p>

<h4 id="the-early-phase-of-the-covid-19-outbreak-in-lombardy-italy"><a href="https://arxiv.org/abs/2003.09320">The early phase of the COVID-19 outbreak in Lombardy, Italy</a></h4>

<h4 id="secondary-attack-rate-and-superspreading-events-for-sars-cov-2"><a href="https://www.thelancet.com/action/showPdf?pii=S0140-6736%2820%2930462-1">Secondary attack rate and superspreading events for SARS-CoV-2</a></h4>
<p>Estimates for some ‘super-spreader’ events in table.</p>

<h4 id="predicting-collapse-of-adaptive-networked-systems-without-knowing-the-network"><a href="https://www.nature.com/articles/s41598-020-57751-y.pdf">Predicting collapse of adaptive networked systems without knowing the network</a></h4>
<p>General on network collapse.</p>

<h4 id="epidemiological-data-from-the-covid-19-outbreak-real-time-case-information"><a href="https://www.nature.com/articles/s41597-020-0448-0.pdf">Epidemiological data from the COVID-19 outbreak, real-time case information</a></h4>

<h4 id="fundamental-principles-of-epidemic-spread-highlight-the-immediate-need-for-large-scale-serological-surveys-to-assess-the-stage-of-the-sars-cov-2-epidemic"><a href="https://www.medrxiv.org/content/10.1101/2020.03.24.20042291v1">Fundamental principles of epidemic spread highlight the immediate need for large-scale serological surveys to assess the stage of the SARS-CoV-2 epidemic</a></h4>

<h4 id="testing-everyone-instead-of-contact-tracing"><a href="https://medium.com/@sten.linnarsson/to-stop-covid-19-test-everyone-373fd80eb03b">Testing everyone instead of contact tracing</a></h4>

<p>if <script type="math/tex">c\ p > (R_0-1)/(R_0-R_q)</script></p>

<p>Then exponential decay of outbreak. <script type="math/tex">R_0</script> ~ 2.4, <script type="math/tex">R_q</script> ~ 0.3 (Wuhan estimate).
c: compliance to self-quarantine after testing positive<br />
p: true positive rate</p>

<p>Fastest test currently: RT-LAMP technology, 10-30 mins, based on RNA detection. Polymerase with reverse transcriptase activity amplifies viral RNA —&gt; H+ release —&gt; pH sensitive dye.
But: it requires 65C incubation (tap water?).
Alternatives:</p>
<ul>
  <li>rolling circle replication</li>
  <li>in vitro viral replication assays</li>
</ul>

<p>Lockdown cost dwarfs testing cost.</p>

<h4 id="imperial-study-on-global-effects"><a href="https://www.imperial.ac.uk/media/imperial-college/medicine/sph/ide/gida-fellowships/Imperial-College-COVID19-Global-Impact-26-03-2020.pdf">Imperial study on global effects</a></h4>

<p>More detail on model. Distributions used for time of next infection.</p>

<h4 id="nn-taleb-tail-risk-of-contagious-diseases"><a href="https://www.researchers.one/media/documents/282-m-corona2.pdf">NN Taleb: Tail Risk of Contagious Diseases</a></h4>

<h4 id="why-everyone-should-wear-a-mask"><a href="https://medium.com/@Cancerwarrior/covid-19-why-we-should-all-wear-masks-there-is-new-scientific-rationale-280e08ceee71">Why everyone should wear a mask</a></h4>

<h4 id="estimates-of-the-severity-of-coronavirus-disease-2019-a-model-based-analysis"><a href="https://www.thelancet.com/journals/laninf/article/PIIS1473-3099(20)30243-7/fulltext">Estimates of the severity of coronavirus disease 2019: a model-based analysis</a></h4>

<h4 id="an-investigation-of-transmission-control-measures-during-the-first-50-days-of-the-covid-19-epidemic-in-china"><a href="https://science.sciencemag.org/content/early/2020/03/30/science.abb6105">An investigation of transmission control measures during the first 50 days of the COVID-19 epidemic in China</a></h4>

<p>Effect of Wuhan travel ban.</p>

<h4 id="predictions-about-peaks"><a href="http://www.healthdata.org/covid/updates">Predictions about peaks</a></h4>
<p>Preds made 05 April. France: 15K deaths by August, UK 66K (!).</p>

<h4 id="preparedness-and-vulnerability-of-african-countries-against-importations-of-covid-19-a-modelling-study"><a href="https://www.thelancet.com/action/showPdf?pii=S0140-6736%2820%2930411-6">Preparedness and vulnerability of African countries against importations of COVID-19: a modelling study</a></h4>

<p>Importation risk from China vs health care system capacity</p>

<h4 id="ron-milo-covid-by-the-numbers"><a href="https://elifesciences.org/articles/57309/">Ron Milo: COVID by the numbers</a></h4>

<p>Estimates on size, mutation rate, genome of virus.
Nucleotide identity with other coronaviruses:</p>
<ul>
  <li>bat CoV: 96%</li>
  <li>pangolin CoV: 91%</li>
  <li>SARS-CoV-1: 80%</li>
  <li>MERS: 55%</li>
  <li>common cold CoV: 50%</li>
</ul>

<p><a href="https://biomedical-engineering-online.biomedcentral.com/track/pdf/10.1186/s12938-017-0370-7">MERS basic reproduction number</a></p>

<p>Authors fit SIR model to data, least-square fit, they find a <script type="math/tex">R_0</script> around 8 (!).
How did they manage to suppress MERS?</p>

<p><a href="https://medium.com/@urialonw/adaptive-cyclic-exit-strategies-from-lockdown-to-suppress-covid-19-and-allow-economic-activity-4900a86b37c7">Uri Alon: Adaptive cyclic-exit-strategies-from-lockdown</a></p>

<p>4 day work+10 day lockdown, antiphased for 2 groups of households.
Review of deterministic, stochastic and also <em>network</em> SEIR model, different cyclic lockdown strategies.</p>

<p>Antiphasing (staggering) uses the timeline of the disease and infectiousness:
<img src="https://miro.medium.com/max/770/0*MWxESwBV56QMF9rc" alt="_config.yml" /></p>

<p><a href="https://github.com/ryansmcgee/seirsplus">SEIRSPLUS: Python package for stochastic and network based SEIR model</a></p>

<p>Python package to set up epidemiologic models easily, used in Uri Alon study.</p>

<h2 id="3-economics-and-politics">3. Economics and politics</h2>

<h4 id="james-meadway-on-novara-16march"><a href="https://novaramedia.com/2020/03/16/coronavirus-will-require-us-to-completely-reshape-the-economy">James Meadway on Novara, 16/march</a></h4>
<p>2008: crisis of banking system. QE prevented a crash
COVID19 more fundamental as it threatens the basis of capitalism, the <em>labour market</em>. Division of labour has to change.
Already trillion USD intervention by Fed.
Epidemics becoming worse bc of climate change.
Sick leave, helicopter money, requisitioning.</p>

<h4 id="jp-morgan-analysis-on-us-small-businesses"><a href="https://institute.jpmorganchase.com/content/dam/jpmc/jpmorgan-chase-and-co/institute/pdf/institute-growth-vitality-cash-flows.pdf">JP Morgan analysis on US small businesses</a></h4>
<p>Most US small businesses have less than 1 month financial reserves if no revenue.</p>

<h4 id="syllabus-of-political-economic-consequences-of-covid19-evgeny-morozov"><a href="https://the-politics-of-covid-19.com/search/?size=n_30_n">Syllabus of political-economic consequences of COVID19 (Evgeny Morozov)</a></h4>

<p>Compilation of more in depth analysis of political and economic consequences of the pandemic.</p>

<h4 id="effects-on-the-global-south"><a href="https://rpalat.wordpress.com/2020/03/27/coronavirus-and-the-world-economy-the-old-is-dead-the-new-cant-be-born/">Effects on the global south</a></h4>

<p>The lacking infrastructure of developing and poor countries to deal with a pandemic.</p>

<h4 id="mike-davis-who-gets-forgotten-in-an-epidemic"><a href="https://www.thenation.com/article/politics/mike-davis-covid-19-essay/">Mike Davis: Who gets forgotten in an epidemic</a></h4>

<h4 id="social-contagion-wuhan-story"><a href="http://chuangcn.org/2020/02/social-contagion/">Social contagion (Wuhan story)</a></h4>

<h4 id="disease-x-who-scenario-2018"><a href="https://edition.cnn.com/2018/03/12/health/disease-x-blueprint-who/index.html">Disease X WHO scenario 2018</a></h4>

<h4 id="m-roberts-war-economy"><a href="https://thenextrecession.wordpress.com/2020/03/30/a-war-economy">M Roberts: War economy?</a></h4>

<p><img src="https://thenextrecession.files.wordpress.com/2020/03/war-4.jpg" alt="_config.yml" /></p>

<h4 id="mike-davis-in-a-plague-year-jacobin-14march"><a href="https://jacobinmag.com/2020/03/mike-davis-coronavirus-outbreak-capitalism-left-international-solidarity">Mike Davis: In a Plague Year (Jacobin 14/March)</a></h4>

<p>Ebola: 1994, African bat cave<br />
SARS: 2003, Guangdong (manufact.)
1918 ‘Spanish flu’ outbreak was H1N1, most fatalities young ppl, maybe older ppl had immunity from flus of 1890s.</p>
<h4 id="monthly-review-covid19-and-circuits-of-capital"><a href="https://monthlyreview.org/2020/04/01/covid-19-and-circuits-of-capital/">Monthly Review: COVID19 and circuits-of-capital</a></h4>

<p>Trump dismantled his pandemic preparation team and left ~700 CDC positions unfilled.
At least 60 percent of novel human pathogens emerge by spilling over from wild animals to local human communities
<a href="https://monthlyreview.org/product/big_farms_make_big_flu/">Big Farms Make Big Flu: Dispatches on Infectious Disease, Agribusiness, and the Nature of Science</a>
<a href="https://linkinghub.elsevier.com/retrieve/pii/S0140673612616845">Colgate/Johnson&amp;Johnson heatmap</a> of where next pandemics might emerge.
But they only talk abt the sources (‘exotic’ food), not demand driving it and networks.
Also growth of slums and incorporating rural communities into urban networks, so forest disease dynamics show up in cities v fast (cf. Mike Davis Planet of Slums). ‘Post-agricultural’ regions.  Deforestation + periurban lack of health infrastructure.
‘Sylvatic pathogens’ (wild animal pathogens) burned out in the forest before now entering susceptible human populations (no healthcare) —&gt; networks of travel/trade.
<a href="https://doi.org/10.1177%2F0020731415611644">Robert G. Wallace et al., Did Neoliberalizing West African Forests Produce a New Niche for Ebola? (2016)</a></p>

<p>‘The lengthier the associated supply chains and the greater the extent of adjunct deforestation, the more diverse (and exotic) the zoonotic pathogens that enter the food chain.
Among recent emergent and reemergent farm and foodborne pathogens, originating from across the anthropogenic domain, are African swine fever, Campylobacter, Cryptosporidium, Cyclospora, Ebola Reston, E. coli O157:H7, foot-and-mouth disease, hepatitis E, Listeria, Nipah virus, Q fever, Salmonella, Vibrio, Yersinia, and a variety of novel inﬂuenza variants, including H1N1 (2009), H1N2v, H3N2v, H5N1, H5N2, H5Nx, H6N1, H7N1, H7N3, H7N7, H7N9, and H9N2.’</p>

<p>Growing genetic monocultures, overcrowding leading to immune suppression.</p>

<p><a href="https://theasiadialogue.com/2020/03/12/the-covid-19-epidemic-and-chinas-wildlife-business-interest/">The COVID-19 epidemic and China’s wildlife business interest</a></p>

<p>1989, the Wildlife Protection Law (WPL)
China’s wildlife farming has evolved into a gigantic business interest. In 2016, the industry produced 520.6 billion yuan (USD 81.3 billion) in revenue. The part of wildlife farming that supplied the exotic food market accounted for 24 per cent (125 billion yuan or USD 19.5 billion) of the total output.</p>

<h4 id="reuters-detailed-account-of-uk-gov-response-timeline"><a href="https://www.reuters.com/article/us-health-coronavirus-britain-path-speci-idUSKBN21P1VF">Reuters detailed account of UK gov response timeline</a></h4>

<h4 id="covid-19-and-the-coming-corporate-debt-catastrophe-13-march"><a href="https://sbhager.com/covid-19-and-the-coming-corporate-debt-catastrophe/">COVID-19 and the Coming Corporate Debt Catastrophe (13, March)</a></h4>

<h4 id="politicoeu-how-europe-failed-the-coronavirus-test"><a href="https://www.politico.eu/article/coronavirus-europe-failed-the-test/">Politico.eu: How Europe failed the coronavirus test</a></h4>
<!---![_config.yml](/images/covid/reaction_eur_states_14978916_1586353616114.png)--->

<p>This is a <a href="https://images.jifo.co/14978916_1586353616114.png">useful table</a> on how many days it took European governments to apply restrictions after the 3rd death from COVID19.
Greece, Eastern European states did it before 3rd death, Spain 9-10 days after, UK, France 2 weeks after.</p>

<p><a href="https://www.sciencemag.org/news/2020/03/modelers-weigh-value-lives-and-lockdown-costs-put-price-covid-19">Can you put a price on COVID-19 options? Experts weigh lives versus economics</a></p>

<p>Summary on modeling efforts to estimate costs of lockdowns vs letting the virus spread.
Early estimates suggested even a year-long lockdown more economically viable.
They made an estimate for economic loss due to deaths.
Important finding: usually fatality rates <em>fall</em> during economic crises (less accidents etc), even if suicides might increase.</p>

<p><a href="https://drive.google.com/file/d/1Vu0wl-9K2dh8MpMqaO85MvE6UH7gcRLx/view">Data Gaps and the Policy Response to the Novel Coronavirus</a></p>

<p>Macroeconomic model to estimate economic effects of lockdowns vs. release.</p>

<p><a href="https://www.kellogg.northwestern.edu/faculty/rebelo/htm/epidemics.pdf">Martin S. Eichenbaum, Sergio Rebelo, Mathias Traband: The Macroeconomics of Epidemics (12 April)</a></p>

<!---!
<div class="infogram-embed" data-id="0991a8d7-ada2-4fc2-a351-f446081b8d95" data-type="interactive" data-title="3 — Comparing when coronavirus measures were implemented"></div><script>!function(e,i,n,s){var t="InfogramEmbeds",d=e.getElementsByTagName("script")[0];if(window[t]&&window[t].initialized)window[t].process&&window[t].process();else if(!e.getElementById(n)){var o=e.createElement("script");o.async=1,o.id=n,o.src="https://e.infogram.com/js/dist/embed-loader-min.js",d.parentNode.insertBefore(o,d)}}(document,0,"infogram-async");</script>
--->

<p><a href="https://climateandcapitalism.com/2020/03/11/capitalist-agriculture-and-covid-19-a-deadly-combination/">Capitalist agriculture and Covid-19: A deadly combination (11 March)</a></p>

  </div>

  <div class="date">
    Written on April  9, 2020
  </div>

  <span>[
  
    
    <a href="/tag/political-economy"><code class="highligher-rouge"><nobr>political-economy</nobr></code>&nbsp;</a>
  
    
    <a href="/tag/globalization"><code class="highligher-rouge"><nobr>globalization</nobr></code>&nbsp;</a>
  
    
    <a href="/tag/epidemics"><code class="highligher-rouge"><nobr>epidemics</nobr></code>&nbsp;</a>
  
    
    <a href="/tag/complex-systems"><code class="highligher-rouge"><nobr>complex-systems</nobr></code>&nbsp;</a>
  
    
    <a href="/tag/ecology"><code class="highligher-rouge"><nobr>ecology</nobr></code>&nbsp;</a>
  
    
    <a href="/tag/covid19"><code class="highligher-rouge"><nobr>covid19</nobr></code>&nbsp;</a>
  
]</span>





<script type="text/javascript" src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<script type="text/x-mathjax-config">
MathJax.Hub.Config({
  TeX: { equationNumbers: { autoNumber: "AMS" } },
  "HTML-CSS": { linebreaks: { automatic: true, width: "75% container"  } },
         SVG: { linebreaks: { automatic: true, width: "75% container"  } }
});
</script>

<script>
       MathJax.Hub.Config({
           config: ["MMLorHTML.js"],
           extensions: ["tex2jax.js","TeX/AMSmath.js","TeX/AMSsymbols.js"],
           jax: ["input/TeX"],
           tex2jax: {
               inlineMath: [ ['$','$'], ["\\(","\\)"] ],
               displayMath: [ ['$$','$$'], ["\\[","\\]"] ],
               processEscapes: false
           },
           TeX: {
               TagSide: "right",
               TagIndent: ".8em",
               MultLineWidth: "85%",
               equationNumbers: {
                  autoNumber: "AMS",
               },
               unicode: {
                  fonts: "STIXGeneral,'Arial Unicode MS'"
               }
           },
           showProcessingMessages: false
       });
</script>


</article>

    </div>

    <div class="wrapper-footer">
      <div class="container">
        <footer class="footer">
          
<a href="mailto:mihaly.koltai@curie.fr"><i class="svg-icon email"></i></a>


<a href="https://github.com/mbkoltai"><i class="svg-icon github"></i></a>

<a href="https://www.linkedin.com/in/mihaly-koltai"><i class="svg-icon linkedin"></i></a>


<a href="https://www.twitter.com/mbkoltai"><i class="svg-icon twitter"></i></a>



        </footer>
      </div>
    </div>

    

  </body>
</html>
